### PR TITLE
Feature/update prettier

### PR DESCRIPTION
**Pull request by garyluu**
Update prettier to latest mainly because of https://github.com/prettier/prettier/issues/7428 , then manually ran prettier on all relevenant files.

The new trailing comma looks weird to me but...it is what it is.

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const wp = require('@cypress/webpack-preprocessor');
 
-module.exports = on => {
+module.exports = (on) => {
   const options = {
-    webpackOptions: require('../../webpack.config')
+    webpackOptions: require('../../webpack.config'),
```
Comment (denis-yuen) [inline cypress/plugins/index.js:5]: You're right, it does look odd

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -38,10 +38,10 @@ describe('AppComponent', () => {
         RouterLinkStubDirective,
         RouterOutletStubComponent,
         TosBannerStubComponent,
-        NotificationStubComponent
+        NotificationStubComponent,
```
Comment (denis-yuen) [inline src/app/app.component.spec.ts:41]: Yep

**Review by denis-yuen — COMMENTED** [review]
Does `--none` https://prettier.io/docs/en/options.html#trailing-commas 
fix the comma?

**Comment by garyluu** [comment]
Adding `--trailing-comma none` to the command does work but...

https://prettier.io/blog/2020/03/21/2.0.0.html first paragraph says they deliberately chose to do that.  As weird as it looks to me, the point of prettier is that it's opinionated so we don't debate on which style to use https://prettier.io/docs/en/option-philosophy.html

**Review by denis-yuen — APPROVED** [review]
ok

**Comment by coverbeck** [comment]
> The new trailing comma looks weird to me but...it is what it is.

Fun fact: IE6 did not allow trailing commas.

File: cypress/plugins/index.js (hunk 1 above)
Comment (coverbeck) [reply cypress/plugins/index.js:5]: The plus is that it makes it easier to add another line, saving you the massive effort of typing in a comma on the previous line. And it does make the diffs a little cleaner. And easier to sort.

But yeah, it still looks weird.

**Review by coverbeck — APPROVED** [review]

**Review by NatalieEO — APPROVED** [review]

**Review by Ldcabansay — APPROVED** [review]

**Pull request merged**